### PR TITLE
Clang tidy fixes 2

### PR DESCRIPTION
**Pull request by kecajjo**

### CODE REVIEW

File: bl/drawable/src/drawable.cpp [hunk 1]
```diff
@@ -9,21 +9,23 @@ void Drawable::addChild(const std::shared_ptr<Drawable> &child, Drawable::Positi
     children[child] = p;
 }
 
-Drawable::Drawable(const Drawable &d) {
-    color = d.color;
-    shape = d.shape;
-    children = d.children;
-}
+void Drawable::removeChild(const std::shared_ptr<Drawable> &child) { children.erase(child); }
```
Comment (ProrokWielki) [inline bl/drawable/src/drawable.cpp]: shouldn't be in this PR it is not fixing any clang-tidy issue

File: bl/drawable/src/drawable.cpp [hunk 2]
```diff
@@ -9,21 +9,23 @@ void Drawable::addChild(const std::shared_ptr<Drawable> &child, Drawable::Positi
...
 
-Drawable::Drawable(Drawable &&d) noexcept {
-    color = d.color;
-    shape = std::move(d.shape);
-    children = std::move(d.children);
-}
+void Drawable::clearChildren() { children.clear(); }
```
Comment (ProrokWielki) [inline bl/drawable/src/drawable.cpp]: shouldn't be in this PR it is not fixing any clang-tidy issue

File: bl/drawable/src/drawable.cpp [hunk 3]
```diff
@@ -9,21 +9,23 @@ void Drawable::addChild(const std::shared_ptr<Drawable> &child, Drawable::Positi
...
+    : shape(std::move(d.shape)), children(std::move(d.children)), color(d.color) {}
 
 Drawable &Drawable::operator=(Drawable &&d) noexcept {
     color = d.color;
     shape = std::move(d.shape);
     children = std::move(d.children);
     return *this;
-}
\ No newline at end of file
+}
```
Comment (ProrokWielki) [inline bl/drawable/src/drawable.cpp]: shouldn't be in this PR it is not fixing any clang-tidy issue

File: bl/drawable/inc/drawable.hpp [hunk 4]
```diff
@@ -29,14 +33,18 @@ class Drawable {
     const std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position> &getChildDrawables()
         const;
     void addChild(const std::shared_ptr<Drawable> &child, Position p);
+    void removeChild(const std::shared_ptr<Drawable> &child);
```
Comment (ProrokWielki) [inline bl/drawable/inc/drawable.hpp]: shouldn't be in this PR it is not fixing any clang-tidy issue

File: bl/drawable/inc/drawable.hpp [hunk 5]
```diff
@@ -3,24 +3,28 @@
 #include <limits>
 #include <list>
 #include <memory>
-#include <tuple>
 #include <unordered_map>
-#include <utility>
+
+// TODO temp struct, will be removed in the next PR
+struct Color {
+    int r, g, b, a;
```
Comment (ProrokWielki) [inline bl/drawable/inc/drawable.hpp]: shouldn't be in this PR it is not fixing any clang-tidy issue

**Review by ProrokWielki — CHANGES_REQUESTED** [review]

File: bl/drawable/inc/drawable.hpp (hunk 5 above)
Comment (kecajjo) [reply bl/drawable/inc/drawable.hpp]: fixed

File: bl/drawable/src/drawable.cpp (hunk 2 above)
Comment (kecajjo) [reply bl/drawable/src/drawable.cpp]: fixed

File: bl/drawable/src/drawable.cpp (hunk 1 above)
Comment (kecajjo) [reply bl/drawable/src/drawable.cpp]: fixed

File: bl/drawable/inc/drawable.hpp (hunk 4 above)
Comment (kecajjo) [reply bl/drawable/inc/drawable.hpp]: fixed

**Review by ProrokWielki — APPROVED** [review]

**Pull request merged**